test/functional/mempool_ephemeral_dust.py

```diff
@@ -47,6 +47,22 @@ def add_output_to_create_multi_result(self, result, output_value=0):
 
         result["new_utxos"].append({"txid": new_txid, "vout": len(result["tx"].vout) - 1, "value": Decimal(output_value) / COIN, "height": 0, "coinbase": False, "confirmations": 0})
 
+    def create_ephemeral_dust_package(self, *, tx_version, dust_tx_fee=0, dust_value=0, num_dust_outputs=1, extra_sponsors=None):
+        """Creates a 1P1C package containing ephemeral dust. By default, the parent transaction
```

Comment:
micro-nit, no need to invalidate acks
```suggestion
        """Creates a 1P1C package containing ephemeral dust by default. By default, the parent transaction
```